doc/release_notes/v2.1.0.rst

```diff
@@ -57,26 +58,25 @@ Changes
 * Reading and adding unknown non-private tags now does not raise an exception
   per default, only when :attr:`~pydicom.config.enforce_valid_values` is set
   (:issue:`1161`)
+* Invalid values used with the :class:`~pydicom.dataset.Dataset`
+  :func:`in<operator.__contains__>` operator now raise :class:`ValueError`
```

Comment:
Maybe should define invalid values here, given #1214., or at least point to [the docs](https://pydicom.github.io/pydicom/stable/reference/generated/pydicom.dataset.Dataset.html ) with new examples there.

Reply:
Changed with #1216 anyway